### ISSUE TITLE
Possible bug in flam3-genome

### ISSUE THREAD

**Issue by roever** [scottdraves/flam3#26]
When you create an parameter file for an animation using flam3-genome with the following command line

env sequence=sheep-in.flam3 nframes=320 loops=0 flam3-genome > sheep-out.flam3

and sheep-in.flam3 contains just 2 flames to interpolate between and one or both contain a tag <symmetry kind=...> then the output for the first and/or last flame flame in the generated sequence in sheep-out.flam3 will contain the xforms that were added to provide the symmetry requested and also the symmetry tag again.

When you then try to render the animation you get rubbish because flam3-animate adds the xforms for the symmetry yet again for the first frame and will try to interpolate it away on the way to the 2nd image resulting in big smears in the first 2 image. Only from the the 3rd image on the results will be right.

Have I used flam3-genome in the wrong way? Or is there a bug in the program?

**Comment by EReckase** [comment]
No, this sounds like it's probably a bug. I'll try to look at it this evening.
